### ISSUE TITLE
Switch to making some status calls based on cloudwatch metrics instead of alarms

### ISSUE THREAD

**Issue by OopsAllErrors** [NASA-IMPACT/hls-status#18]
Currently, the code determines Green/Yellow/Red status by making API calls to determine if certain alarms are OK/Insufficient Data/In Alarm. This can be more accurate, because we are already pulling metrics which are more up-to-date than the alarms which are run periodically. I think those metrics are normally only pulled when the "Details" button is pressed, so this may require us to change when metrics are pulled. Please make the following changes:

For S30:
- For the status "Produced Granules w/in Expected Range"
  - Set to status Red if Granules Succeeded is less than 5,000 within the prior 24 hours
  - Set to status Yellow if Granules Succeeded is less than 7,000 within the prior 24 hours
  - Otherwise, set to status Green

- For the status "Nominal % Errors"
  - Determine an appropriate metric (using the existing alarm) to pull this information from.
  - Set to status Red if metric indicates >20% of S30 granules failed within the prior 24 hours **or if no data is available**
  - Set to status Yellow if metric indicates >10% but <20% of S30 granules failed within the prior 24 hours
  - Otherwise, set to status Green

For L30:
Basically, same as above. We will need to use different thresholds for L30, but I don't know what those should be yet. For now, use the same values as above, don't be surprised if the status seems off.
